https module not found

```
{ Error: Cannot find module './https'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/lewis/dev/cirrus/node_modules/dd-trace/src/plugins/index.js:9:12)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3) code: 'MODULE_NOT_FOUND' }
```

It seems that [src/plugins/index.js](https://github.com/DataDog/dd-trace-js/blob/master/src/plugins/index.js) is attempting to require `src/plugins/https.js` which does not exist.

--- a/src/plugins/index.js
+++ b/src/plugins/index.js
@@ -6,7 +6,6 @@ module.exports = {
   'express': require('./express'),
   'graphql': require('./graphql'),
   'http': require('./http'),
-  'https': require('./https'),
   'mongodb-core': require('./mongodb-core'),
   'mysql': require('./mysql'),
   'mysql2': require('./mysql2'),

